Backend/dev/migrations-read-only/provider-dashboard/API_Management_Driver.sql

```diff
@@ -197,3 +197,21 @@ UPDATE atlas_bpp_dashboard.transaction
 UPDATE atlas_bpp_dashboard.transaction
   SET endpoint = 'PROVIDER_MANAGEMENT/DRIVER/POST_DRIVER_SYNC_DOC_AADHAR_PAN'
   WHERE endpoint = 'DriverAPI PostDriverSyncDocAadharPanEndpoint';
+
+-- {"api":"PostDriverPersonId","migration":"endpointV2","param":null,"schema":"atlas_bpp_dashboard"}
+UPDATE atlas_bpp_dashboard.transaction
+  SET endpoint = 'ProviderManagementAPI_DriverAPI_PostDriverPersonIdEndpoint'
+  WHERE endpoint = 'DriverAPI PostDriverpersonIdEndpoint';
+
+
+------- SQL updates -------
+
+-- {"api":"PostDriverUpdateVehicleManufacturing","migration":"endpoint","param":"DriverAPI UpdateVehicleManufacturingEndPoint","schema":"atlas_bpp_dashboard"}
+UPDATE atlas_bpp_dashboard.transaction
+  SET endpoint = 'ProviderManagementAPI_DriverAPI_PostDriverUpdateVehicleManufacturingEndpoint'
+  WHERE endpoint = 'DriverAPI UpdateVehicleManufacturingEndPoint';
+
+-- {"api":"PostDriverUpdateVehicleManufacturing","migration":"endpointV2","param":null,"schema":"atlas_bpp_dashboard"}
+UPDATE atlas_bpp_dashboard.transaction
+  SET endpoint = 'ProviderManagementAPI_DriverAPI_PostDriverUpdateVehicleManufacturingEndpoint'
+  WHERE endpoint = 'DriverAPI PostDriverUpdateVehicleManufacturingEndpoint';
```

Comment:
Why these queries?

Reply:
@roman-bodavskiy Its a new API, is it always generated? Also first query looks irrelevant to this change